week-1/errors/1.js

```diff
@@ -1,4 +1,5 @@
 // trying to create an age variable and then reassign the value by 1
 
-const age = 33;
+let age = 33;
 age = age + 1;
+console.log(age);
```

Comment:
Always try to remember to clean up any extra console.log statements if they aren't required by the instructions